### PR TITLE
feat: add CrossFacilityReport schema

### PR DESCRIPTION
**Pull request by marianakozyra**

### CODE REVIEW

File: schema/CrossFacilityReport.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                },
+                "visited": {
+                    "type": [ "string", "null" ]
+                }
+            },
+            "required": [
+                "facility_name",
+                "region_name",
+                "mrn",
+                "unit",
```
Comment (mjurenka) [inline schema/CrossFacilityReport.json]: this is marked as optional few lines above

File: schema/CrossFacilityReport.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                "room",
+                "rank_today",
+                "rank_yesterday",
```
Comment (mjurenka) [inline schema/CrossFacilityReport.json]: rank yesterday is definitely an optional field

**Review by mjurenka — CHANGES_REQUESTED** [review]

**Comment by mjurenka** [comment]
@VanesSaiva please take a look at this

**Review by matusblhs — APPROVED** [review]

**Review by mjurenka — APPROVED** [review]
approved

**Pull request merged**